src/Entity/DTO/MeshConceptDTO.php

```diff
@@ -14,107 +15,79 @@
 class MeshConceptDTO
 {
     /**
-     * @var int
      * @IS\Id
      * @IS\Expose
      * @IS\Type("string")
      */
-    public $id;
+    public string $id;
 
     /**
-     * @var string
-     *
      * @IS\Expose
      * @IS\Type("string")
      */
-    public $name;
+    public string $name;
 
     /**
-     * @var bool
-     *
      * @IS\Expose
      * @IS\Type("boolean")
      */
-    public $preferred;
+    public bool $preferred;
 
     /**
-     * @var string
-     *
      * @IS\Expose
      * @IS\Type("string")
      */
-    public $scopeNote;
+    public string $scopeNote;
 
     /**
-     * @var string
-     *
      * @IS\Expose
      * @IS\Type("string")
      */
-    public $casn1Name;
+    public string $casn1Name;
 
     /**
-     * @var string
-     *
      * @IS\Expose
      * @IS\Type("string")
      */
-    public $registryNumber;
+    public string $registryNumber;
```

Comment:
should be optional `?string $registryNumber`.

Reply:
✔️